src/renderer/components/drawer/drawer.tsx

```diff
@@ -108,7 +108,7 @@ class NonInjectedDrawer extends React.Component<DrawerProps & Dependencies & typ
     if (ev.detail === 3) {
       const selection = window.getSelection();
 
-      if (selection?.anchorNode?.parentNode) {
+      if (selection?.anchorNode?.parentNode && !(selection.anchorNode instanceof HTMLLabelElement)) {
```

Comment:
Not really related but do we really have global click handler which counts for clicks? 
Do you know what is the reasoning behind it? Sounds horrible. 

About the fix itself: sure. Code could explain the reasoning better, e.g. by introducing variable with good name